chalk-parse/src/parser.lalrpop

```diff
@@ -204,7 +204,6 @@ TyWithoutFor: Ty = {
     },
     <n:Id> "<" <a:Comma<Parameter>> ">" => Ty::Apply { name: n, args: a },
     <p:ProjectionTy> => Ty::Projection { proj: p },
-    <a:AliasTy> => Ty::Alias { alias: a },
```

Comment:
why did you remove this?

Reply:
I accidentally left it in from the merge above!  It was removed in #324 [here](https://github.com/rust-lang/chalk/pull/324/files#diff-6fa8bab6c97bead41db277caab4db7a2L193) and is gone on master now